### PR TITLE
disable wand builder by default

### PR DESCRIPTION
**Pull request by inkhey**
related to #235

### CODE REVIEW

File: preview_generator/preview/builder_factory.py [hunk 1]
```diff
@@ -98,7 +98,12 @@ def load_builders(self, force: bool = False) -> None:
             from preview_generator.preview.generic_preview import PreviewBuilder  # nopep8
 
             for cls in get_subclasses_recursively(PreviewBuilder):
-                self.register_builder(cls)
+                if cls.__name__ == "ImagePreviewBuilderWand":
+                    self.logger.warn(
+                        "ImagePreviewBuilderWand builder deprecated, do not register by default."
+                    )
```
Comment (raphj) [inline preview_generator/preview/builder_factory.py:104]: Does this mean that out of the box on a new installation, PreviewGenerator will output a warning a deprecated builder?
I think a warning should be output to the user if they can do something to fix it.

```suggestion
                    self.logger.info(
                        "ImagePreviewBuilderWand builder is deprecated and is not registered by default. Consider using ImagePreviewBuilderIMConvert instead"
                    )
```

Comment (inkhey) [reply preview_generator/preview/builder_factory.py:104]: I don't have strong opinon on this.

Comment (lebouquetin) [reply preview_generator/preview/builder_factory.py:104]: @inkhey I do not see reason not to load Wand: you are about to potentially implement regression in user implementations based on preview-generator

File: preview_generator/preview/builder/image__wand.py [hunk 2]
```diff
@@ -15,22 +14,11 @@
 from preview_generator.utils import imagemagick_supported_mimes
 
 
-def convert_pdf_to_jpeg(pdf: typing.IO[bytes], preview_size: ImgDims) -> BytesIO:
```
Comment (lebouquetin) [inline preview_generator/preview/builder/image__wand.py:18]: Why do you remove this code? Wand allow to convert pdf to jpeg, why do you remove this feature?

Comment (inkhey) [reply preview_generator/preview/builder/image__wand.py:18]: This code is not removed, this code has been moved to pdf builder.

File: preview_generator/preview/builder_factory.py (hunk 1 above)
Comment (inkhey) [reply preview_generator/preview/builder_factory.py:104]: after discussion with @lebouquetin, we make clear that not loading wand is not a problem, as all supported format by wand are supported by imagemagick too.

**Review by raphj — APPROVED** [review]

**Pull request merged**